README.md

```diff
@@ -120,16 +120,18 @@ $TOOLCHAIN/usr/bin/swift build --build-tests
 $TOOLCHAIN/usr/bin/swift test
```

Comment:
I recommend clipping this out as well - if you're reading straight down, it looks like the stock `swift test` is expected to work, even with detailing more the the DYLD details just below.

Reply:
Great catch, thank you!